src/radio/styles/radio.m.css

```diff
@@ -2,78 +2,33 @@
 
 .root {
 	display: block;
-	font: var(--base-font);
-	margin-bottom: var(--line-spacing);
-	padding: 0.25em 0 0.25em 2em;
 	position: relative;
 }
 
-.input {
-	opacity: 0;
-	position: absolute;
-	z-index: -1;
-}
-
-.inputWrapper {
-	left: 0;
-	position: absolute;
-	top: 0;
-}
-
-.inputWrapper::before,
-.inputWrapper::after {
-	border-radius: 50%;
-	box-sizing: border-box;
-	content: '';
-	display: block;
-}
-
-.inputWrapper::before {
-	border: 2px solid var(--input-highlight-color);
-	height: 1.5em;
-	width: 1.5em;
-}
-
-.inputWrapper::after {
-	background-color: var(--input-highlight-color);
-	height: calc(1.5em - 6px);
-	left: 3px;
-	position: absolute;
-	top: 3px;
-	transform: scale(0);
-	transition: all var(--short-animation-duration) var(--short-animation-easing);
-	width: calc(1.5em - 6px);
-}
-
-/* checked style */
-.checked .inputWrapper::after {
-	transform: scale(1);
-}
-
-/* disabled and readonly styles */
-.disabled .inputWrapper::before,
-.readonly .inputWrapper::before {
-	border-color: var(--input-border-color);
-}
+.input {}
```

Comment:
Just clarifying, these empty rulesets have to be defined so they can be themed, right?